### DISCUSSION TITLE
Is it possible to adjust bitrate downward?

### DISCUSSION THREAD

**Discussion by doctorttt** [blakeblackshear/frigate#6334] [General]
Hi, I have been scratching my head for a couple day to see whether bitrate could be adjusted. I researched extensively on google and on github (i.e. `output_args`), and I'm surprised that no one asked questions related to bitrate. Can someone please help?

Please see screenshot. The output file is at 8,193kbps bitrate, making a 60-second video like 60MB large. Is it possible to set a lower bitrate or even set variable bitrate? I looked into `output_args` but couldn't find anything. 

Bitrate adjustment is fundamental in NVR. Please point me to the right direction. Thank you

![output](https://user-images.githubusercontent.com/132242705/235394758-516aa138-8539-41f4-9224-32c39f1bd3de.jpg)

**Comment by blakeblackshear** [comment]
It's best to do this on the camera. To do this in the NVR, you will have to decode and re-encode and that will require quite a bit of resources. Frigate just runs ffmpeg, so it's possible, but probably not a good idea unless you have a good GPU.

**Reply by doctorttt** [reply]
Thank you Blake.  Just a follow up. If I do have a good gpu, how do I decode and re-encode to make the recordings smaller?

**Reply by blakeblackshear** [reply]
You could transcode in go2rtc like this: https://github.com/AlexxIT/go2rtc#source-ffmpeg

Or you could modify the output parameters for the record role in frigate. The parameters would be dependent on the hardware acceleration you were using. You will most likely need to sacrifice quality and/or compression to do it realtime at any decent resolution.

**Comment by doctorttt** [comment]
Just wanted to close this one out. I looked into my old Synology, and while Synology's Surveillance Station can set bitrate on the recordings, it's in fact actually setting the bitrate on the camera, which can also be done by going into the camera's settings directly. I'm now able to use Frigate to record at the bitrate I want. Overall, I agree with Blake. It's best to set the bitrate, etc. in in the camera.
